app/models/ext_management_system.rb

```diff
@@ -694,6 +694,10 @@ def conversion_hosts
     host_conversion_hosts + vm_conversion_hosts
   end
 
+  def total_active_tasks
+    host_conversion_hosts.sum(:total_active_tasks) + vm_conversion_hosts.sum(:total_active_tasks)
+  end
+
```

Comment:
This just points out the difficulty I'm going to have trying to move the V2V model code - specifically the conversion_hosts model - out of the core repo and into the V2V repo.  Still open to suggestions on how this can be done...